app/javascript/tests/patient/Patient.test.js

```diff
@@ -398,7 +398,7 @@ describe('Patient', () => {
     const section = wrapper.find('#case-information');
```

Comment:
commenting here because it relates to the patient component.  I realize this was me not you, but I noticed a styling discrepancy with the edit link styling on the review page versus the monitoree details

my suggestion is changing [this part ](https://github.com/SaraAlert/SaraAlert/blob/1100-exposure-editable-from-isolation/app/javascript/components/patient/Patient.js#L62) of the conditional.  I'd remove the surrounding div, and update the button className to "edit-link p-0"